spec/models/document_type_spec.rb

```diff
@@ -12,7 +12,19 @@
         validator = JSON::Validator.fully_validate(document_type_schema, document_type)
         expect(validator).to(
           be_empty,
-          "Validation for #{document_type['id']} failed: \n\t#{validator.join("\n\t")}",
+          "Document type validation for #{document_type['id']} failed: \n\t#{validator.join("\n\t")}",
+        )
+      end
+    end
+
+    it "should have locale keys that conform to the document type locale schema" do
```

Comment:
Since we're expecting to have this for a third time soon, perhaps it's time for a helper?

Reply:
Done 👍